packages/runtime/src/widgets/Collapsible.tsx

```diff
@@ -71,9 +75,79 @@ export type CollapsibleProps = {
 } & EnsembleWidgetProps &
   HasItemTemplate;
 
+const EvaluateItem = React.memo(
+  ({
+    context,
+    template,
+    currentIndex,
+    setEvaluated,
+  }: {
+    context: object;
+    currentIndex: number;
+    template: CollapsibleItem;
+    setEvaluated: React.Dispatch<React.SetStateAction<CollapsibleTemplate[]>>;
+  }): null => {
+    const evaluatedData = useEvaluate({ ...template }, { context });
```

Comment:
This should evaluate the key, otherwise you will create additional bindings for label and children attributes you don't need.